apps/docusaurus/docs/guides/keyless-accounts.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Aptos Keyless Integration Guide"
+---
+
+Aptos Keyless allows your users to set up an Aptos blockchain account from their existing Google accounts, rather than from a traditional secret key or mnemonic. In a nutshell, with Aptos Keyless, a user’s blockchain account is their Google account. In the future, Aptos Keyless will support many OpenID Connect (OIDC) providers, not just Google.
+
+Importantly, Aptos Keyless maintains user privacy in two ways:
+
+1. A user’s email address is not revealed on-chain to anybody, including other users and validators.
+2. A user’s blockchain address and associated transaction history is hidden from the identity provider (e.g. Google).
+
+Keyless accounts are revolutionary to users for the following reasons:
+
+1. “1-click” account creation via familiar Web2 logins like `Sign In with Google`.
+2. Ability to transact on the Aptos blockchain without needing to navigate away from the application experience to download a wallet.
+3. Requires no secret key management by the user. This means blockchain account access is synonymous with access to one’s OIDC account and Web2-like recovery flows are available to regain access to one’s blockchain account in case the user ever loses access to their OIDC account.
+4. Seamless cross-devices experiences; users log in with their OIDC account no matter what device they are on - no need to download wallet software on each device, import their keys and encrypt them with a password, which must be maintained.
+
+:::tip Keyless Account Scoping
+Use of the **_Aptos Keyless Integration Guide_** will allow for the integration of keyless accounts directly into your application. This means that blockchain accounts created on your application are scoped to the domain of your application (logging in with your Google account on dApp A and logging in with your Google account on dApp B will create separate accounts). Stay tuned for more to come on Aptos’ plan to allow Keyless accounts to be used portably across applications.
```

Comment:
Nit: 
This means that blockchain accounts created on your application are scoped to the domain of your application
This means that blockchain accounts are scoped to your application's domain